docker/docker-compose/build.yaml

```diff
@@ -0,0 +1,35 @@
+# This is a replace for run_docker.sh
+version: '2.3'
+services:
+  radameon_build:
+    environment:
+      - OPER_HOME=/home/oper
+      - HUID
+      - HGID
+      - DISPLAY
+      - QT_X11_NO_MITSHM=1
+
+    image: "${_docker_namespace}/build_${_compiler}:${_image_tag}"
```

Comment:
Does namespace actually has to be parametrized? It seems to be pretty useless, this is not abstract enough to fit any scenario, so we can pretty much use the namespace we defined during build.